src/pages/sponsors_inventory/popup/meta-field-values.js

```diff
@@ -21,15 +21,14 @@ const MetaFieldValues = ({
   handleFieldValueChange,
   handleRemoveValue,
   handleAddValue,
-  entity,
-  setEntity
+  formik
```

Comment:
use hook dont pass formik drilling it down
check doc
https://formik.org/docs/api/useFormikContext